src/main/java/com/minecolonies/colony/Structures.java

```diff
@@ -92,14 +101,14 @@ private static void loadStyleMaps(Path basePath) throws IOException
                 String style = path.getParent().getFileName().toString();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'style' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)